src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -1,35 +1,79 @@
-import { Link, Skeleton, Typography } from '@mui/material';
+import { Box, Link, Skeleton, Typography } from '@mui/material';
 import { useQuery } from '@tanstack/react-query';
 import { useTranslation } from 'react-i18next';
+import { fetchPerson } from '../../../api/cristinApi';
 import { fetchRegistration } from '../../../api/registrationApi';
-import { Registration, RegistrationStatus } from '../../../types/registration.types';
+import { ProfilePicture } from '../../../components/ProfilePicture';
+import { PublicationNote, Registration, RegistrationStatus } from '../../../types/registration.types';
+import { getIdentifierFromId } from '../../../utils/general-helpers';
+import { getFullCristinName } from '../../../utils/user-helpers';
 import { StyledStatusMessageBox } from '../../messages/components/PublishingRequestMessagesColumn';
 
 interface DeletedRegistrationInformationProps {
   registration: Registration;
+  unpublishingNote: PublicationNote;
 }
 
-export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+function extractId(createdBy: string) {
+  return '/cristin/person/' + createdBy.split('@')[0];
+}
+
+export const DeletedRegistrationInformation = ({
+  registration,
+  unpublishingNote,
+}: DeletedRegistrationInformationProps) => {
   const { t } = useTranslation();
+
+  const duplicateRegistrationIdentifier = getIdentifierFromId(registration.duplicateOf ?? '');
+
   const duplicateRegistrationQuery = useQuery({
-    enabled: !!registration.duplicateOf,
-    queryKey: ['registration', registration.duplicateOf],
-    queryFn: () => fetchRegistration(registration.duplicateOf ?? ''),
+    enabled: !!duplicateRegistrationIdentifier,
+    queryKey: ['registration', duplicateRegistrationIdentifier],
+    queryFn: () => fetchRegistration(duplicateRegistrationIdentifier),
     meta: { errorMessage: t('feedback.error.get_registration') },
   });
 
+  const cristinIdentifier = unpublishingNote.createdBy ? extractId(unpublishingNote.createdBy) : null;
+  const personQuery = useQuery({
+    queryKey: ['cristinPerson', cristinIdentifier],
```

Comment:
Andre plasser vi bruker `fetchPerson` ser ut til å ha identifier som key, så den må være samme her for å kunne ta nytte av cachede kall. Tror vi gjorde en feil i starten, ved å tvinge alle keys til å ha en prefiks (som du har lagt inn her), men tenker det er en annen sak for en annen dag 😇 

```suggestion
    queryKey: [cristinIdentifier],
```